Repository/RevisionRepository.php

```diff
@@ -609,6 +602,46 @@ public function lockRevisions(ContentType $contentType, \DateTime $until, $by, $
         return $qbUpdate->getQuery()->execute();
     }
 
+    public function lockAllRevisions(\DateTime $until, string $by): int
+    {
+        return $this->lockRevisions(null, $until, $by, true, false);
+    }
+
+    public function unlockRevisions(?ContentType $contentType, string $by): int
```

Comment:
nullable parameters should be at the end

Reply:
Unfortunately the function was already use somewhere else before my refactor (my bad!)